### PR TITLE
feat(packages): Add wizer

### PR DESCRIPTION
**Pull request by blaine-arcjet**
This adds [Wizer](https://github.com/bytecodealliance/wizer/tree/v4.0.0) in the same fashion as Bytecode Alliance's `wasm-tools`.

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

### CODE REVIEW

**Comment by ajayk** [comment]
Thanks @blaine-arcjet for your contributions 🎉

File: wizer.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      mv target/release/wizer ${{targets.destdir}}/usr/bin/
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: bytecodealliance/wizer
+    strip-prefix: v
+    tag-filter: v
```
Comment (ajayk) [inline wizer.yaml:39]:
```suggestion
    tag-filter: v

test:
  pipeline:
    - runs: |
       wizer -V

```

Comment (ajayk) [reply wizer.yaml:39]: needs a basic test something like above will do

**Comment by blaine-arcjet** [comment]
Thanks for the quick review @ajayk 🤗

**Review by ajayk — APPROVED** [review]

**Pull request merged**